pyinfra/api/host.py

```diff
@@ -1,9 +1,14 @@
+import typing as t
```

Comment:
For consistency with the rest of the code please import directly ie `from typing import TYPE_CHECKING`.